docs/pages/choose-an-edition/teleport-team.mdx

```diff
@@ -39,194 +41,250 @@ web & tsh--Client traffic-->proxy
+[Choose an Edition](introduction.mdx).
 
 </Notice>
 
-## Get started with Teleport Team
+This guide explains how to register a local server with a
+Teleport Team account. After you register the server, you can access it through the 
+Web UI in a browser or using the terminal. You can also record your sessions, so you 
+can review it later.
 
-This guide walks you through the process of registering a local server with a
-Teleport Team account. You can then access the server through the browser and
-the terminal, and record your session so you can review it later.
-
-### Prerequisites
+## Prerequisites
 
 - A Teleport Team account. You can sign up for a free trial at the following
   page:
 
   https://goteleport.com/signup/
 
-- Docker installed on your workstation. We will use a Docker container to
-  register a server with Teleport via the Teleport SSH Service. You can find
-  installation instructions on [Docker's
-  website](https://docs.docker.com/get-docker/).
+  After you sign up, you receive an email invitation to activate your account.
+  Use this account to authenticate your identity when you access the Teleport
+  Team tenant.
 
-  <Notice type="tip">
+- Docker installed on your workstation. 
 
-  Docker is only required for the local demo environment we use in this guide.
-  If you want to get started registering your servers in your infrastructure
-  with Teleport, view our documentation on how to [register servers and
-  configure access to them](../server-access/introduction.mdx).
+  This guide illustrates how to register a server with Teleport Team using a Docker 
+  and the Teleport SSH Service. Docker is only required for the local demo environment 
+  used in this guide. You can find installation instructions for Docker on [Docker's
+  website](https://docs.docker.com/get-docker/). If you want to register servers in 
+  Teleport Team without using Docker, see 
+  [register servers and configure access to them](../server-access/introduction.mdx).
 
-  </Notice>
+- The `tsh` client tool. 
+  
+  The `tsh` client tool is only required to access the server from a terminal. 
+  For installation instructions, see [Installation Guide](../installation.mdx). 
+  If you don't install he `tsh` client tool, you can access the server in 
+  Teleport Team using the Web UI through your browser.
 
-- The `tsh` client tool. Visit our [Installation Guide](../installation.mdx)
-  for instructions. If you want to get started quickly, you can skip this for
-  now and access Teleport through your browser.
+## Step 1/5. Spin up a server
 
-### Step 1/4. Spin up your server
+To spin up a new server using Docker:
 
-Start a Docker container on your workstation to register as a server in your
-Teleport Team tenant:
+1. Open a terminal shell on your workstation.
 
-```code
-$ docker run -it ubuntu:22.10 /bin/bash
-```
+1. Start a Docker container on your workstation to prepare a server that you want 
+register in your Teleport Team tenant:
```

Comment:
```suggestion
enroll with your Teleport Team tenant:
```

Reply:
"enroll with" sounds weird to me. How about "enroll as a resource in your Teleport Team tenant."